ErgoLux/localization/strings.ru-RU.resx

```diff
@@ -402,7 +402,7 @@
     <comment/>
   </data>
   <data name="strLblPlotWindow" xml:space="preserve">
-    <value>Окно участка (секунды)</value>
+    <value>График участка (секунды)</value>
```

Comment:
Not sure about this one. This string "Plot window (seconds)" doesn't refer to a type of plot. Instead, it refers to the plots' abscissa axis range (that's the "window"). The default value is 20 seconds and, as data is aquired, the axis range changes from 0-20 to 1-21, 2-22, and so on.

This is the screenshot for this settings section:
![imagen](https://user-images.githubusercontent.com/63915486/229425733-51185f1b-f948-4cfc-aa37-fe12d54a22a3.png)

And that's the translated one:
![imagen](https://user-images.githubusercontent.com/63915486/229425638-0e694177-c5ef-4895-8103-85ec79d2a05c.png)